### PR TITLE
[#305] Add Windows CI pipeline

### LINKED ISSUE #305: Add Windows platform to CI
Some users have been reporting bugs in Windows, mostly due to path handling and path separators.
It shouldn't be a problem to support Windows in the server, but if we want to do so we need to add Windows as a platform to our CI pipelines to avoid regressions. I don't even know if this is a possibility in Travis or something else.

### PR DESCRIPTION
**Pull request by robertoaloi**
### Description

There are still a few issues with the pipeline:

* The pre-compile hook fails
* rebar3 detects the error, but terminates with a zero status code

Fixes #305 .

### CODE REVIEW

**Comment by alanz** [comment]
Is this still valid/relevant?

**Comment by robertoaloi** [comment]
> Is this still valid/relevant?

I think so. A Windows pipeline would help avoid regressions for users.

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -56,3 +56,22 @@ jobs:
...
+      uses: actions/checkout@v2
+    - name: Install Erlang
+      run: choco install -y erlang --version ${{ matrix.otp-version }}
+    - name: Install rebar3
+      run: choco install -y rebar3 --version 3.13.1
+    - name: Compile
+      run: rebar3 compile
+    - name: Run CT Tests
+      run: rebar3 ct
+    - name: Run PropEr Tests
```
Comment (TheGeorge) [inline .github/workflows/build.yml:76]: should we upload the logs for debugging?

**Comment by jfacorro** [comment]
Nice! I will use this for clojerl!

**Pull request merged**